packages/app/pages/checkout/[network]/[address].tsx

```diff
@@ -10,16 +10,34 @@ import Web3 from "web3";
 import passportFactoryJson from "@cabindao/nft-passport-contracts/artifacts/contracts/PassportFactory.sol/PassportFactory.json";
 import passportJson from "@cabindao/nft-passport-contracts/artifacts/contracts/Passport.sol/Passport.json";
 import { styled } from "../../../stitches.config";
-import { Button } from "@cabindao/topo";
 import BN from "bn.js";
 import { useCallback, useEffect, useRef, useState } from "react";
 import axios from "axios";
+import { IpfsImage } from "../..";
```

Comment:
let's move this into its own `/components` file. We don't want pages importing from each other

Reply:
done